Formula/d/di.rb

```diff
@@ -21,8 +21,22 @@ class Di < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "8d3898fd49a626edd4b27777a5b5f9a58c2e9f65fd4939eed0a3e47d0ee6a361"
   end
 
+  depends_on "cmake" => :build
+  depends_on "pkgconf" => :build
+
   def install
-    system "make", "install", "prefix=#{prefix}"
+    args = %W[
+      -DDI_BUILD:STATIC=Release
+      -DDI_VERSION:STATIC=#{version}
+      -DDI_LIBVERSION:STATIC=#{version}
```

Comment:
I think we can just remove the type hints
```suggestion
      -DDI_BUILD=Release
      -DDI_VERSION=#{version}
      -DDI_LIBVERSION=#{version}
```